models/user.js

```diff
@@ -0,0 +1,23 @@
+import mongoose from "mongoose";
+const { Schema } = mongoose;
+
+const userSchema = new Schema({
+    username: {
+        type: String,
```

Comment:
There should be a min length on usernames. We can't enforce that here with passwords since they are hashes.

**Username AND Password Length should be enforced on the front end upon sign up.**